server/game/cards/attachments/02/lady.js

```diff
@@ -1,47 +1,60 @@
 const DrawCard = require('../../../drawcard.js');
 
 class Lady extends DrawCard {
+    constructor(owner, cardData) {
+        super(owner, cardData);
+        this.oldOwner = null;
```

Comment:
This initialization (and the one on Nymeria) isn't strictly necessary. It does let the reader of the code know to expect this variable later but doesn't really fit with other places we've used similar patterns.